lib/src/context/room_context.dart

```diff
@@ -198,6 +199,9 @@ class RoomContext extends ChangeNotifier with ChatContextMixin {
   /// Get the [Room] instance.
   Room get room => _room;
 
+  ///
+  final bool enableAudioVisulizer;
```

Comment:
fill in the comment here - and make sure to cover the comments i made in https://github.com/livekit/client-sdk-flutter/pull/638#discussion_r1867066294